changelog/1.7.0.md

```diff
@@ -1,21 +1,66 @@
 # Scala.meta 1.7.0
 
+## Cross-compilation against Scala.js
+
+It is our great pleasure to announce that the overwhelming majority of scala.meta modules are now supported by Scala.js!
+In 1.7.0, it is possible to do advanced tasks that include parsing, quasiquoting and prettyprinting in a browser.
+The only unsupported modules are `scalahost` and `testkit`.
+
+## Parser for Scaladoc comments
+
+## Semantic API
+
+- `scalahost` has been split into `scalahost`, a library that provides a scalac-based implementation of `Mirror`,
+and `scalahost-nsc`, a compiler plugin that uses this library to emit semantic databases from within scalac.
+Offline mirrors underwent a breaking change to accommodate this refactoring (see below).
+
+- Semantic databases are no longer emitted in textual format, and are now encoded using protocol buffers.
+Textual format remains an important debugging and prototyping vehicle, so prettyprinting still works like it used to in 1.6.0.
+
+- Compiler messages, including warnings and infos, are now included in semantic databases.
+This curious feature has been added in order to improve integration with `scalafmt`.
+We're looking forward to featuring more semantic information in our databases in the future!
+
+## `contrib`
+
+The `contrib` module introduced in 1.6.0 gained a lot of traction.
+Lots of experimentation and discussion are going on in `contrib`!
+
+- The `Extract[A, B]` typeclass has been added for convenient extraction of Stats/Mods/Annotations/Defs.
+  - See the `contrib` testsuite for more example usages.
+  - More use-cases for Extract are in the pipeline, follow #741 for more details.
+- `structurally` and `syntactically` have been added as enrichments to `Set[Tree]`.
+  - See the `contrib` testsuite for examples.
```

Comment:
@DavidDudson @olafurpg Do you think we should mention more new developments in this area?